mscore/preferences.h

```diff
@@ -27,6 +27,8 @@
 
 namespace Ms {
 
+extern QString mscoreGlobalShare;
```

Comment:
OK I tested. Because it's needed in fluid and zerberus... Maybe we should then put it there and not in preferences.h